src/libraries/System.Security.Cryptography.Pkcs/tests/Pkcs12/Pkcs12Documents.cs

```diff
@@ -112,7 +112,7 @@ internal static class Pkcs12Documents
         // orapki wallet create -wallet ewallet.p12
         // mkstore -wrl ewallet.p12 -createUserCredential a b c d
         // mkstore -wrl ewallet.p12 -createCredential a_prod_db a_test_user
-        //  (prompted secret: "potatos are tasty")
```

Comment:
This typo is also hidden in the hex blob below. You would need to regenerate the blob to fix it.

Reply:
How ?

Reply:
Using the tools in the comment.

I do not think it is worth fixing this typo. It might have been intentional - people often use misspellings in passwords.

Reply:
Thanks. It makes sense. I will discard that change.